debian/rules

```diff
@@ -39,12 +39,12 @@ clean:
 	# Add here commands to clean up after the build process.
 	#$(MAKE) clean
 
-	dh_clean 
+	dh_clean
 
 install: build
 	dh_testdir
 	dh_testroot
-	dh_clean -k 
+	dh_clean -k
```

Comment:
We should split formatting changes into a new pr then rebase, all these tiny changes make it hard to review

Reply:
I can see about splitting the formatting changes off. Shouldn't be too difficult.